packages/next-on-pages/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@cloudflare/next-on-pages",
-	"version": "1.0.2",
+	"version": "1.0.3",
```

Comment:
The same version needs to be applied to the `eslint-plugin-next-on-pages` package

Reply:
It is applied now 👍